users/models.py

```diff
@@ -112,6 +112,22 @@ def get_role_for_organization(self, organization_id):
 
         return organization_user.role
 
+    def is_user_org_admin(self, organization_id: int, return_role: bool = False):
```

Comment:
maybe can rename it to `is_org_admin`?
Because this method will always be called on an instance of User no?